extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeStreamExtractor.java

```diff
@@ -66,16 +65,25 @@ public DateWrapper getUploadDate() throws ParsingException {
  
     @Override
     public String getThumbnailUrl() throws ParsingException {
-        return baseUrl + JsonUtils.getString(json, "thumbnailPath");
+        return baseUrl + JsonUtils.getString(json, "previewPath");
     }
 
     @Override
     public String getDescription() throws ParsingException {
+        String description = "";
+        Downloader dl = NewPipe.getDownloader();
```

Comment:
We might want to check whether we have the full description before starting another download.
I think it should be save to check whether the description ends with `"..."`. I think the description's max length is 250 characters. But this should be validated.

Reply:
Done. See commit message, 255 ≤ shortened descriptions length's ≤ 269.

Reply:
Tested in app and it doesn't work. I will change this later.